src/libraries/Error.sol

```diff
@@ -102,6 +102,9 @@ library Error {
     /// @dev is thrown if contract call is invalid (for axelar)
     error NOT_APPROVED_BY_GATEWAY();
 
+    /// @dev is thrown if all message bridges fail
+    error MESSAGE_BRIDGE_FAILURE();
```

Comment:
nit: rename to be more descriptive `MESSAGE_SEND_FAILED()` or `MULTI_MESSAGE_SEND_FAILED()`